javascript/search/binary_search.js

```diff
@@ -0,0 +1,19 @@
+// This function should be implemented, array is assumed to be sorted
+function binarySearch(array, searchItem) {
+    var present = array.indexOf(searchItem);
+    if (present == -1) {
+        throw "Item not present in array";
```

Comment:
Usually in JS you'll want to use the error class, like so: `throw new Error('Item not present in array');`